osm2gtfs/creators/et_addisababa/config.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
All config should come with some regression tests, so we can make sure our future changes in osm2gtfs do not break your own implementation.

Reply:
I will probably not be able to add those from my current limited python skills. Help is appreciated.